perfkitbenchmarker/linux_benchmarks/cluster_boot_benchmark.py

```diff
@@ -83,7 +83,26 @@ def Run(benchmark_spec):
   samples = vm_util.RunThreaded(_GetTimeToBoot, params)
   logging.info(samples)
   assert len(samples) == len(vms)
-  return samples
+  total_boot = benchmark_spec.vm_boot_endtime - benchmark_spec.vm_boot_starttime
```

Comment:
This total time includes things that are not included in the individual machine boot times. As it's currently written, this captures the entire RunThreaded(benchmark_spec.PrepareVm, ...) call. If you check the definition of [PrepareVm](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/benchmark_spec.py#L276), it calls vm.Create(), vm.AllowRemoteAccessPorts(), vm.WaitForBootCompletion(), and then a bunch of other things including vm.AddMetadata(). For individual VMs, the boot time measurement starts partway through [vm.Create()](https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/resource.py#L127), within the call to vm._CreateResource(), which comes after vm.CreateDependencies(). The boot time measurement ends partway through vm.WaitForBootCompletion().
So the difference is that this total time measurement contains the time to create dependencies (e.g. Network objects) and also the time to do post-creation steps (e.g. adding metadata).